core/templates/pages/exploration-player-page/layout-directives/audio-bar.component.spec.ts

```diff
@@ -0,0 +1,568 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the 'License');
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS-IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for for AudioBarComponent.
+ */
+
+import { AudioBarComponent } from 'pages/exploration-player-page/layout-directives/audio-bar.component';
+import { Voiceover } from 'domain/exploration/voiceover.model';
+import { AssetsBackendApiService } from 'services/assets-backend-api.service';
+import { AudioBarStatusService } from 'services/audio-bar-status.service';
+import { AudioPlayerService } from 'services/audio-player.service';
+import { AutogeneratedAudioPlayerService } from 'services/autogenerated-audio-player.service';
+import { AudioPreloaderService } from '../services/audio-preloader.service';
+import { AudioTranslationLanguageService } from '../services/audio-translation-language.service';
+import { AudioTranslationManagerService } from '../services/audio-translation-manager.service';
+import { PlayerPositionService } from '../services/player-position.service';
+import { ComponentFixture, discardPeriodicTasks, fakeAsync, flush, TestBed, tick, waitForAsync } from '@angular/core/testing';
+import { EventEmitter, NO_ERRORS_SCHEMA, Pipe } from '@angular/core';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+
+@Pipe({name: 'translate'})
+class MockTranslatePipe {
```

Comment:
I believe we have 1 defined in the test utils file

Reply:
updated!